src/components/bixicard.js

```diff
@@ -35,13 +42,14 @@ export default class BixiCard extends React.Component {
             return res.json();
         })
         .then(res => {
+            let stationArray = [];
             res.stations.filter(function(station){
-                // milton/clark station 6209
-                return station.n === "6209";
+                return stationList.indexOf(station.n) > -1;
             }).forEach(function(station){
-                // set state of bikes available
-                scope.setState({bikesAvailable: station.ba});
+                stationArray.push(station);
             });
+            scope.setState({stationData: stationArray});
+            this.cycleBixi();
```

Comment:
My thinking initially with cycleBixi() was that the timing of the display cycle could be separate (and more frequent than) the interval for the API call. 
Also, next I'm thinking it could be good to figure out a way to synchronize the interval with the language switching interval.

Let me know what you think of the changes? Thanks!